docs/StardustDocs/topics/schemas.md

```diff
@@ -11,248 +11,33 @@ It ignores order of columns in [`DataFrame`](DataFrame.md), but tracks column hi
 
 In Jupyter environment compile-time [`DataFrame`](DataFrame.md) schema is synchronized with real-time data after every cell execution.
 
-In IDEA projects, you can use the [Gradle plugin](gradle.md#configuration) to extract schema from the dataset
+In IDEA projects, you can use the [Gradle plugin](schemasGradle.md#configuration) to extract schema from the dataset
 and generate extension properties.
 
-## DataSchema workflow in Jupyter
 
-After execution of cell
+## Popular use cases with Data Schemas
 
-<!---FUN createDfNullable-->
+Here's a list of the most popular use cases with Data Schemas.
 
-```kotlin
-val df = dataFrameOf("name", "age")(
-    "Alice", 15,
-    "Bob", null
-)
-```
+* [**Data Schemas in Gradle projects**](schemasGradle.md) <br/>
+  If you are developing a server application and building it with Gradle.
 
-<!---END-->
+* [**DataSchema workflow in Jupyter**](schemasJupyter.md) <br/>
+  If you prefer Notebooks.
 
-the following actions take place:
+* [**Schema inheritance**](schemasInheritance.md) <br/>
+  It's worth knowing how to reuse Data Schemas generated earlier.
 
-1. Columns in `df` are analyzed to extract data schema
-2. Empty interface with [`DataSchema`](schema.md) annotation is generated:
+* [**Custom Data Schemas**](schemasCustom.md) <br/> 
+  Sometimes it is necessary to create your own scheme.
```

Comment:
*schema

Reply:
https://dictionary.cambridge.org/dictionary/english/scheme

Reply:
Yes, but we call it "schema" everywhere else. Also "scheme" means something different than "schema". A scheme is a plan, an idea. A schema is a written down or drawn-out idea. So schema is rightfully used across the docs.